src/main/java/core/basesyntax/dao/FruitShopDao.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.dao;
+
+import java.nio.file.Path;
+import java.util.List;
+
+public interface FruitShopDao {
```

Comment:
* interface by the look of it's methods has nothing to do with the fruits so far, it jsut reads and writes files, so it should be named correctly
```suggestion
public interface FileService {
```
* overall combining this sort of logic can be possible, but it's better to separate this into 2 interfaces: `FileReader` & `FileWriter` as we are practicing the **SOLID** principles in this HW, and we need t keep up with that `S`